README.md

```diff
@@ -330,6 +331,65 @@ end
 
 5. Your app is now able to access the authenticated `ProductsController` from the `HomeController` using session tokens.
 
+### Marking shops as uninstalled on app/uninstalled webhook
+
+To ensure OAuth still works with session tokens, your app must update its shop records in the event a shop uninstalls your app. You can be notified of these events by subscribing to the `app/uninstall` [webhook](https://shopify.dev/docs/admin-api/rest/reference/events/webhook).
```

Comment:
```suggestion
To ensure OAuth continues to work with session tokens, your app must update its shop records in the event a shop uninstalls your app. The app can receive notifications of these events by subscribing to the `app/uninstall` [webhook](https://shopify.dev/docs/admin-api/rest/reference/events/webhook).
```

Reply:
Would it be worth giving an example of a scenario where OAuth and session tokens need to work together?

Maybe this README is not the right place for it. But I wonder if the connections/distinctions between OAuth, Session tokens/JWT, and cookies is clear. Something to consider explaining further in shopify.dev docs.

Reply:
Yeah that makes sense to add that in the `shopify.dev` guides.